### PR TITLE
#154894833 - Updates readme to include install instructions

### PR DESCRIPTION
**Pull request by altereagle**
* These install instructions should work for anyone who is brand new.
* Can you test it out on your local machine for me, and if there is anything missing lets add it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,62 @@
...
+  * If you want to know more about Node, [this video is a great start!](https://youtu.be/U8XF6AFGqlc)
+  * You can [try writing JavaScript running as a Node process here!](https://repl.it/repls/SophisticatedFaintGnatcatcher)
+
+#### <span style="color: maroon;">&#11041; Redis</span> - [redis.io](https://redis.io/)
+<span style="color: maroon;">Redis stores, retrieves, and communicates between Node proceseses. It performs these tasks very reliably and quicky! A Node process is JavaScript code that is run using Node. *Arc uses Redis to monitor performance, store data, and send messages between Node proceses.*</span>
+  * You can [try Redis here!](http://try.redis.io/)
+
+#### Getting Started: Install Arc
+> (You should also have [git](https://git-scm.com/) installed on your machine)
+
```
Comment (tammycole) [inline README.md]: Are default settings ok here? 
image

Comment (altereagle) [reply README.md]: Yep!

File: README.md [hunk 2]
```diff
@@ -1,24 +1,62 @@
...
+##### 1. Download this repo using `git` in your terminal
```
Comment (tammycole) [inline README.md]: Maybe a basic command line tutorial here? 
cd Desktop
cd Projects

File: README.md [hunk 3]
```diff
@@ -1,24 +1,62 @@
...
+```bash
+git clone git@github.com:altereagle/arc.git
```
Comment (tammycole) [inline README.md]: image

File: README.md [hunk 4]
```diff
@@ -1,24 +1,62 @@
...
+##### 2. Navigate to the directory of the repo that git cloned
+
+```bash
+cd arc
+```
+
+##### 3. Install node modules
+```bash
+npm install
+```
```
Comment (tammycole) [inline README.md]: image

Comment (tammycole) [reply README.md]: https://docs.npmjs.com/cli/install

Not sure what I should do here...

Comment (altereagle) [reply README.md]: You might need to install Node. npm is installed when you install node.

File: README.md (hunk 3 above)
Comment (altereagle) [reply README.md]: Ah, you did an http install instead of a ssh install.  That works without setting up the SSH stuff.

**Comment by altereagle** [comment]
Code climate is bugging out. I am going to merge this and see if it starts behaving better.  I will create a new PR if more changes need to be made.

**Pull request merged**